cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/PackCommandTest.java

```diff
@@ -359,4 +359,29 @@ public void testPackTemplatePackageWithACompilerPackageDependency() throws IOExc
         Assert.assertFalse(mainBalContent.contains("public function newFunctionByCodeModifiermain(string params) " +
                 "returns error? {\n}"));
     }
+
+    @Test(description = "Pack a ballerina project with a constraint package specified")
```

Comment:
Shall we add a test case to check how it behaves if the given constraint package is not available?